server/documents/modules/dropdown.html.eco

```diff
@@ -3058,6 +3058,11 @@ themes      : ['Default', 'GitHub', 'Material']
           <td>false<div class="ui label">New in 2.2.13</div></td>
           <td>Whether values with matching cases should be treated as identical when adding them to a dropdown.</td>
         </tr>
+        <tr>
+          <td>ignoreSearchCase</td>
+          <td>false<div class="ui label">New in 2.8.0</div></td>
```

Comment:
The default is `true` instead of `false`

Reply:
🙈